### PR TITLE
Creating shell script for single job (vs hp) + fixing setup.py

### PR DESCRIPTION
**Pull request by fprost**
Thank you!

### CODE REVIEW

File: experiments/tf_trainer/keras_gru_attention/run.hp.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (nthain) [inline experiments/tf_trainer/keras_gru_attention/run.hp.sh]: Is run.hp for hyper parameter tuning? Let's rename this file to make that more clear. Also a one line comment at the top would be great. I'm planning on going back to add similar comments to the other run scripts.

File: experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh [hunk 2]
```diff
@@ -12,11 +12,12 @@ gsutil acl set private ${REMOTE_COMET_API_KEY_FILE} &&
 gcloud ml-engine jobs submit training tf_trainer_${MODEL_NAME}_${USER}_${DATETIME} \
```
Comment (nthain) [inline experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh]: Similarly a one line comment at the top stating that this dosn't do hyperparameter tuning and to use the other file instead.

File: experiments/tf_trainer/keras_gru_attention/run.hp.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    --verbosity=debug \
+    --config="tf_trainer/${MODEL_NAME}/hparam_config.yaml" \
+    -- \
+    --train_path="${GCS_RESOURCES}/toxicity_q42017_train.tfrecord" \
+    --validate_path="${GCS_RESOURCES}/toxicity_q42017_validate.tfrecord" \
+    --embeddings_path="${GCS_RESOURCES}/glove.6B/glove.6B.300d.txt" \
+    --model_dir="${JOB_DIR}/model_dir" \
+    --comet_key_file="${REMOTE_COMET_API_KEY_FILE}" \
+    --comet_team_name="jigsaw" \
+    --comet_project_name="experiments${USER}"
```
Comment (nthain) [inline experiments/tf_trainer/keras_gru_attention/run.hp.sh]: This is a better system. At some point, I'd like to move to include the dataset in the job_dir and job name as well (and probably in comet). But for now this looks good.

File: experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh (hunk 2 above)
Comment (fprost) [reply experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh]: done

File: experiments/tf_trainer/keras_gru_attention/run.hp.sh (hunk 1 above)
Comment (fprost) [reply experiments/tf_trainer/keras_gru_attention/run.hp.sh]: done

File: experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh (hunk 2 above)
Comment (fprost) [reply experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh]: done

File: experiments/tf_trainer/keras_gru_attention/run.hp.sh (hunk 3 above)
Comment (fprost) [reply experiments/tf_trainer/keras_gru_attention/run.hp.sh]: done

**Pull request merged**